libcbor/PKGBUILD

```diff
@@ -0,0 +1,43 @@
+# Contributer: Reza Tavakoli <rta.0070@gmail.com>
+
+pkgbase="libcbor"
+pkgname=("${pkgbase}" "${pkgbase}-devel")
+pkgver=0.9.0
+pkgrel=1
+pkgdesc="A C library for parsing and generating CBOR, a general-purpose schema-less binary data format"
+arch=('i686' 'x86_64')
+url="https://github.com/PJK/libcbor"
+license=('MIT')
+makedepends=("cmake" "make" "gcc")
+source=(https://github.com/PJK/libcbor/archive/v${pkgver}/${pkgname}-${pkgver}.tar.gz)
+sha256sums=('da81e4f9333e0086d4e2745183c7052f04ecc4dbcffcf910029df24f103c15d1')
+
+build() {
+  cd ${pkgname}-${pkgver}
+  cmake . \
+    -Bbuild \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_INSTALL_PREFIX=/usr \
+    -DCBOR_CUSTOM_ALLOC=ON \
+    -DBUILD_SHARED_LIBS=ON 
+  make -C build
+  make -C build DESTDIR=${srcdir}/dest install
```

Comment:
you may want to consider making these `cmake --build build` calls instead of make to adhere to a user set `CMAKE_GENERATOR`